automated/linux/ota-apps/z-99-aklite-callback.toml

```diff
@@ -0,0 +1,5 @@
+[bootloader]
+reboot_command = "/bin/true"
+[pacman]
+callback_program = "/var/sota/aklite-callback.sh"
+
```

Comment:
Can this file be the same all across the different ota-rollback, ota-update and ota-apps?

Reply:
Same as with aklite-callback.sh. This is a shared file, but I don't know how to use it in all test dirs.